cli/sync_complete.go

```diff
@@ -0,0 +1,43 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/agent/agentsocket"
+	"github.com/coder/serpent"
+)
+
+func (r *RootCmd) syncComplete() *serpent.Command {
+	return &serpent.Command{
+		Use:   "complete <unit>",
+		Short: "Signal that a service has finished",
+		Long:  "Mark a service unit as complete, indicating it has finished its work and is ready. This allows dependent units that are waiting for this unit to proceed with their startup. Call this after a service has completed its startup and is ready to accept connections or requests.",
```

Comment:
Nit: Terminology:

- Short vs Long using vastly different wording. Mark vs signal. finished vs complete/finished/ready, etc.
- Service (familiarity), here it's unclear whether we're talking about startup scripts, programs started by those scripts, etc. And especially, doesn't help the reader understand the difference.